community/examples/hpc-slurm-gromacs.yaml

```diff
@@ -68,12 +69,6 @@ deployment_groups:
                 projections:
                   all: '{name}/{version}-{compiler.name}-{compiler.version}'
       commands: |
-        # Un-comment and update mirror_url to install from spack cache
-        # if ! spack mirror list | grep -q gcs_cache; then
-        #   spack mirror add --scope site gcs_cache gs://optionally_set_spack_cache_bucket
-        # fi
-        # spack buildcache keys --install --trust
-
```

Comment:
I would leave this. Was there a reason to remove it?

Reply:
Added back.